first_challenge.exs

```diff
@@ -0,0 +1,42 @@
+{limit1, _} = IO.gets("Enter the first number: ") |> Integer.parse
+{limit2, _} = IO.gets("Enter the second number: ") |> Integer.parse
+
+defmodule Calculatesequence do
+  def sequence(limit,number) when limit > 0 do
+    number = "#{limit}" <> number
+    sequence(limit-1,number)
+  end
+
+  def sequence(_limit,number) do
+    number
+  end
+end
+
+defmodule Math do
+  def isDivisor(n,cont) do
+    cond do
+      rem(n, 3) == 0 -> _cont = cont + 1
+      true -> cont
+    end
```

Comment:
Very good by separating this logic on external functions and modules, what about renaming this function to something more accurate such as `sum_if_divisible_by_three` also you can avoid using` _cont` assignation because you are returning count on the same line